skimage/measure/_regionprops.py

```diff
@@ -194,7 +195,7 @@ def coords(self):
                           for i in range(self._ndim)]).T
 
     @only2d
-    def eccentricity(self):
+    def eccentricity(self): #suppress warning
```

Comment:
@holly-c-gibbs I think you meant to use this comment as a to-do. The line underneath should also be silenced.